### PR TITLE
dev: update sql admin syntax

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? 

Update `admin` syntax for TiDB 4.0.

### What is the related PR or file link(s)? 

- [x] Reference link(s):
    * https://github.com/pingcap/tidb/pull/12484

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v4.0 (TiDB 4.0 versions)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [x] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

File: dev/reference/sql/statements/admin.md [hunk 1]
```diff
@@ -73,6 +77,40 @@ admin show ddl jobs;
 +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
 ```
 
+```sql
+-- 查看正在执行的 DDL 任务已经最近 5 条已经执行完的 DDL 任务。
```
Comment (AilinKid) [inline dev/reference/sql/statements/admin.md]:
```suggestion
-- 查看正在执行的 DDL 任务中最近 5 条已经执行完的 DDL 任务。
```

File: dev/reference/sql/statements/admin.md [hunk 2]
```diff
@@ -52,6 +55,7 @@ ADMIN CHECK TABLE tbl_name [, tbl_name] ...;
 {{< copyable "sql" >}}
 
 ```sql
+-- 查看正在执行的 DDL 任务已经最近 10 条已经完成的 DDL 任务。未指定 `NUM` 时，默认只显示最近 10 条已经执行完的 DDL 任务。
```
Comment (AilinKid) [inline dev/reference/sql/statements/admin.md]:
```suggestion
-- 查看正在执行的 DDL 任务中最近 10 条已经完成的 DDL 任务。未指定 `NUM` 时，默认只显示最近 10 条已经执行完的 DDL 任务。
```

**Comment by crazycs520** [comment]
@lilin90 PTAL

File: dev/reference/sql/statements/admin.md [hunk 3]
```diff
@@ -73,6 +77,40 @@ admin show ddl jobs;
 +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
 ```
 
+```sql
+-- 查看正在执行的 DDL 任务中最近 5 条已经执行完的 DDL 任务。
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]: Same with the above one. Please move out this line.

File: dev/reference/sql/statements/admin.md [hunk 4]
```diff
@@ -73,6 +77,40 @@ admin show ddl jobs;
...
+| 42     | test    | t1         | drop index    | none                 | 32        | 37       | 0         | 2019-01-10 12:34:35.204 +0800 CST | synced        |
+| 41     | test    | t1         | add index     | public               | 32        | 37       | 0         | 2019-01-10 12:33:22.62 +0800 CST  | synced        |
+| 40     | test    | t1         | drop column   | none                 | 32        | 37       | 0         | 2019-01-10 12:33:08.212 +0800 CST | synced        |
++--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
+```
+
+{{< copyable "sql" >}}
+
+```sql
+-- 查看 test 数据库中未执行完成的 DDL 任务，包括正在执行中以及最近 5 条已经执行完但是执行失败的 DDL 任务。
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]: ditto

**Comment by bb7133** [comment]
PTAL @lilin90

File: dev/reference/sql/statements/admin.md [hunk 5]
```diff
@@ -55,6 +58,8 @@ ADMIN REPAIR TABLE tbl_name CREATE TABLE STATEMENT;
 
 ## 使用示例
 
+查看正在执行的 DDL 任务中最近 10 条已经完成的 DDL 任务。未指定 `NUM` 时，默认只显示最近 10 条已经执行完的 DDL 任务。
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
执行以下命令，可查看正在执行的 DDL 任务中最近 10 条已经完成的 DDL 任务。未指定 `NUM` 时，默认只显示最近 10 条已经执行完的 DDL 任务。
```

File: dev/reference/sql/statements/admin.md [hunk 6]
```diff
@@ -79,6 +84,44 @@ admin show ddl jobs;
 +--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
 ```
 
+查看正在执行的 DDL 任务中最近 5 条已经执行完的 DDL 任务。
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
执行以下命令，可查看正在执行的 DDL 任务中最近 5 条已经执行完的 DDL 任务：
```

File: dev/reference/sql/statements/admin.md [hunk 7]
```diff
@@ -79,6 +84,44 @@ admin show ddl jobs;
...
+| 45     | test    | t1         | add index     | write reorganization | 32        | 37       | 0         | 2019-01-10 12:38:36.501 +0800 CST | running       |
+| 44     | test    | t1         | add index     | none                 | 32        | 37       | 0         | 2019-01-10 12:36:55.18 +0800 CST  | rollback done |
+| 43     | test    | t1         | add index     | public               | 32        | 37       | 6         | 2019-01-10 12:35:13.66 +0800 CST  | synced        |
+| 42     | test    | t1         | drop index    | none                 | 32        | 37       | 0         | 2019-01-10 12:34:35.204 +0800 CST | synced        |
+| 41     | test    | t1         | add index     | public               | 32        | 37       | 0         | 2019-01-10 12:33:22.62 +0800 CST  | synced        |
+| 40     | test    | t1         | drop column   | none                 | 32        | 37       | 0         | 2019-01-10 12:33:08.212 +0800 CST | synced        |
++--------+---------+------------+---------------+----------------------+-----------+----------+-----------+-----------------------------------+---------------+
+```
+
+查看 test 数据库中未执行完成的 DDL 任务，包括正在执行中以及最近 5 条已经执行完但是执行失败的 DDL 任务。
```
Comment (lilin90) [inline dev/reference/sql/statements/admin.md]:
```suggestion
执行以下命令，可查看 test 数据库中未执行完成的 DDL 任务，包括正在执行中以及最近 5 条已经执行完但是执行失败的 DDL 任务。
```

**Pull request merged**